src/utils/convertToMoment.js

```diff
@@ -7,18 +7,18 @@ export default (newProps, momentProps) => {
     momentProps.forEach(key => {
         const value = newProps[key];
 
-        switch (R.type(value)) {
-            case 'Undefined':
-                return;
-            case 'Null':
-                dest[key] = null;
-                return;
-            default:
-                dest[key] = moment(value);
-        }
+        if (value === null || value === undefined) {
+            dest[key] = null;
+
+            if (key === 'initial_visible_month') {
+                dest[key] = moment();
```

Comment:
So apparently an invalid moment value is not good enough for `initial_visible_month`, doing moment() on the other hand uses the current month..